dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -1,3 +1,6 @@
+# CACHEKEY: "bionic_coq-V2018-05-07-V2"
```

Comment:
What is the point of repeating the CACHEKEY here? Unless this isn't actually a real comment...

Reply:
The point is to allow a script to check that this version here is in sync with what's inside `.gitlab-ci.yml`

Reply:
For automatic checking we should be able to check that any commit modifying the dockerfile also modifies the cachekey in gitlab.yml.

Reply:
That's also a valid way to do it I guess. Also cool would be to autogenerate `.gitlab-ci.yml` and `Dockerfile` using a promote rule.